docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Using Docker to run garage
+
+Currently there are three types of garage images:
+  - ci: used for the continuous integration of garage in TravisCI and not
+    intended for final users.
+  - headless: garage without environment visualization.
+  - NVIDIA: garage with environment visualization using an NVIDIA graphics
+    card.
+
+
+## Headless image
+
+To run an example launcher in the container, execute:
+```
+docker run -it --rm rlworkgroup/garage-headless python examples/tf/trpo_cartpole.py
+```
+
+This image does not come with a MuJoCo key, so if you want to run environments
+using MuJoCo, the environment variable `MJKEY` has to be set with the
+corresponding content of the key file in your file system:
+```
+# New line characters in MuJoCo key files (they're in DOS format) are
+# replaced by '\n' to pass the file as a command argument.
+export MJKEY="$(sed ':a;N;$!ba;s/\r\r//g;s/\n/\\n/g' <your_key_path>)"
+```
+
+Then pass the variable `MJKEY` to the container in the docker-run command:
+```
+docker run \
+  -it \
+  --rm \
+  -e MJKEY="${MJKEY}" \
+  rlworkgroup/garage-headless python examples/theano/trpo_swimmer.py
+```
+
+For every release of garage there's a version of this container, which you can
+specify through the release number. For example:
+```
+docker run --rm rlworkgroup/garage-headless:2018.10 python examples/tf/trpo_cartpole.py
```

Comment:
nah

Reply:
?

Reply:
we aren't adding containers for the previous release, so this tag won't exist yet.